src/coreclr/jit/rationalize.cpp

```diff
@@ -469,9 +393,11 @@ Compiler::fgWalkResult Rationalizer::RewriteNode(GenTree** useEdge, Compiler::Ge
             RewriteAssignment(use);
             break;
 
-        case GT_IND:
         case GT_BLK:
-            RewriteIndir(use);
+            if (varTypeIsSIMD(node))
```

Comment:
What's creating BLK nodes of TYP_SIMD still? I thought your other PR should have generally avoided those?